app/controllers/phrasing_phrases_controller.rb

```diff
@@ -16,7 +16,7 @@ def help
 
   def index
     params[:locale] ||= I18n.default_locale
-    @phrasing_phrases = PhrasingPhrase.fuzzy_search(params[:search], params[:locale])
+    phrasing_phrases
```

Comment:
set this as:
```ruby
@phrasing_phrases = phrasing_phrases
```

Then you could don't need to set the instance var in the method. I prefer this way as it's more explicit.

Reply:
Done